privacy.md

```diff
@@ -0,0 +1,57 @@
+# Blue Blocker Privacy Policy
+
+This privacy policy ("policy") will help you understand how Blue Blocker collects, uses, and protexts the data you provide when you use the Blue Blocker extension/addon (herein: "the Extension").
+
+We reserve the right to change this policy at any given time, and will do our best to promptly update you on changes. If you want to make sure that you are up to date with the latest changes, we advise you to visit the version of this policy hosted on [GitHub](https://github.com/kheina-com/Blue-Blocker/blob/main/privacy.md).
+
+## What User Data the Extension Collects
+
+-   Session cookies set by the Twitter web client.
+-   CSRF tokens used in requests to the Twitter API.
+-   The URL of any request made by the Twitter web client to the Twitter API. These URLs may include the URLs of pages that you visit.
+-   Data sent to the Twitter web client by Twitter's servers ("Twitter data".)
+
+## Why the Extension Collects Your Data
+
+-   Session cookies are used to authenticate requests to the blocking/muting endpoint of the Twitter API.
+-   CSRF tokens are used to allow requests to be sent and processed by Twitter.
+-   URLs are used to filter responses and to parse responses in order to extract Twitter user objects.
+-   Twitter data is used to determine whether a Twitter user should be blocked, and to properly form requests to the blocking/muting endpoint of the Twitter API.
+    -   Twitter usernames, display names, and user IDs contained in this data are additionally used to create a history of users that have been blocked by the extension, and generate a list of users to not block in the future.
+    -   This data may also be used to allow other extensions or addons to add additional checks in determining if the user should be blocked.
+
+**Your data will never be used for any commericial purposes.**
+
+## What User Data is Stored by the Extension
+
+We are comitted to safeguarding all data that you provide to the Extension. We have done everything in our power to prevent data theft, unauthorized access, and disclosure by storing as much data locally as is feasible, and only using well understood browser APIs to store your data.
```

Comment:
```suggestion
We are committed to safeguarding all data that you provide to the Extension. We have done everything in our power to prevent data theft, unauthorized access, and disclosure by storing as much data locally as is feasible, and only using well understood browser APIs to store your data.
```